microsite/blog/2023-12-13-backstage-and-optic.mdx

```diff
@@ -0,0 +1,208 @@
+# Validating OpenAPI schemas with test traffic using Optic
+
+Hey all, today I'm going to be talking about Backstage's use of [Optic's](https://github.com/opticdev/optic) `capture` command, why I think you should use it too, and why it's really cool. Most of you reading this probably don't know me, so let me introduce myself. I'm Aramis, maintainer of Backstage's OpenAPI tooling project area, I've been working for the past year on improving the schema-first developer experience inside of Backstage.
+
+## What is schema-first development?
+
+Schema-first development is using a schema, ie GraphQL, OpenAPI, gRPC, etc, to drive server and front end client implementation. It requires developers to be comfortable with the schema and expects developers to start any changes at the schema instead of at the router or client layer.
```

Comment:
might start by explaining the problem schema first dev solves. You could even name the section "## What problem we were trying to solve" and the last lines are about schema first

Reply:
updated in https://github.com/sennyeya/backstage/commit/49078cb39b521a38430357f635d654d7007ac055